packages/main-nav/src/NavContainer.tsx

```diff
@@ -45,6 +45,7 @@ export function NavContainer({ variant, children }: NavContainerProps) {
 			theme={theme}
 			background={background}
 			color="text"
+			fontFamily="body"
```

Comment:
I don' think this should go on the container? What problem are you trying to solve?

Reply:
Maybe we want and optional `applyTypographyDefaults` prop on `Core`.

Reply:
Good point. I'll put it on the list item